src/routes/docs/products/functions/develop/+page.markdoc

```diff
@@ -468,8 +468,8 @@ Explore the request object with the following function, which logs all request p
 {% multicode %}
 ```js
 export default async ({ req, res, log }) => {
-    log(req.bodyRaw);                     // Raw request body, contains request data
-    log(JSON.stringify(req.body));        // Object from parsed JSON request body, otherwise string
+    log(req.bodyJson);                     // Raw request body, contains request data
+    log(JSON.stringify(req.bodyJson));        // Object from parsed JSON request body, otherwise string
```

Comment:
Fix the alignment of the comments. Please make sure to address the other code samples too.